pipelines/controller/src/main/java/com/google/fhir/analytics/DataProperties.java

```diff
@@ -63,64 +63,94 @@ public class DataProperties {
     Default.String.class, Default.Integer.class, Default.Boolean.class, Default.Long.class
   };
 
+  @SuppressWarnings("NullAway.Init")
   private FhirFetchMode fhirFetchMode;
 
+  @SuppressWarnings("NullAway")
```

Comment:
What is the difference between `@SuppressWarnings("NullAway")` and `@SuppressWarnings("NullAway.Init")`? Why do we need one here but the other above?